app/src/main/java/org/schabi/newpipe/util/ShareUtils.java

```diff
@@ -21,22 +22,80 @@ private ShareUtils() {
      * Open the url with the system default browser.
      * <p>
      * If no browser is set as default, fallbacks to
-     * {@link ShareUtils#openInDefaultApp(Context, String)}
+     * {@link ShareUtils#openInDefaultApp(Context, Intent)}
+     *
+     * @param context                the context to use
+     * @param url                    the url to browse
+     * @param httpDefaultBrowserTest the boolean to set if the
+     *                               test for the default browser will be for HTTP protocol
+     *                               or for the created intent
+     */
+    public static void openUrlInBrowser(final Context context, final String url,
+                                        final Boolean httpDefaultBrowserTest) {
```

Comment:
Why `Boolean` and not `boolean`?

Reply:
Oh, I didn't know this (I am not a developer). I will change it.